scripts/ci/ci.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
@domenkozar I suspect GH is lying, but confirm just in case. Do we have this in `master`?

Reply:
yeah